mod/wikirate/set/type/wikirate_company.rb

```diff
@@ -13,10 +13,3 @@ def indirect_contributor_search_args
   ]
 end
 
-format :html do
-  def edit_slot args
-    # refer to claim.rb
-    super args.merge( :core_edit=>true )
-  end
-end
```

Comment:
this makes you edit within the structure of the page. we're much better off now without this!